pod.go

```diff
@@ -242,11 +242,11 @@ type PodConfig struct {
 	NetworkModel  NetworkModel
 	NetworkConfig NetworkConfig
 
-	// Rootfs is the pod root file system in the host.
-	// This can be left empty if we only have a set of containers
-	// workload images and expect the agent to aggregate them into
-	// a pod from the guest.
-	RootFs string
+	// Interactive specifies if the pod runs in the foreground.
+	Interactive bool
+
+	// Console is a console path provided by the caller.
+	Console string
```

Comment:
Ah yes sorry, I have made lots of different changes and I didn't split the commits very well.